npm Release v1.0.1 is missing the `dist` folder

Hello!

First of, thank you for this package, it's working really well for my use case and I'm glad to have it.

As part of routine dependency upgrades, I wanted to update to the latest v1.0.1, but I get the following error:

```
Error: Cannot find module 'pg-error-enum' or its corresponding type declarations.
```

And indeed, there seems to be a problem with the release on npm, because it's missing the `dist` folder: https://www.npmjs.com/package/pg-error-enum/v/1.0.1?activeTab=code

![Image](https://github.com/user-attachments/assets/7a21ace9-7471-4d0b-8a19-f14e342c2009)

Downgrading obviously fixes it, but I hope you can fix it on your end as well. Thank you in advance.

--- a/package.json
+++ b/package.json
@@ -26,6 +26,7 @@
     "lint:check": "eslint",
     "format:check": "prettier . --check",
     "format:write": "prettier . --write",
+    "prepublishOnly": "npm run build",
     "test": "node bin/test.js"
   },
   "files": [

